src/hooks/programAreas/useProgramAreas.ts

```
@@ -0,0 +1,17 @@
+import { getProgramAreasByIds } from "@/data/firestore/programAreas";
+import { ProgramArea } from "@/types/scheduling/schedulingTypes";
+import { useQuery, useQueryClient } from "@tanstack/react-query";
+
+export default function useProgramAreas(ids: string[]) {
+  const queryClient = useQueryClient();
+  return useQuery({
+    queryKey: ['programAreas', ids],
+    queryFn: async () => {
+      const queryIds = ids.filter(id => !queryClient.getQueryData(['programAreas', id]));
+      const programAreas = await getProgramAreasByIds(queryIds);
+      programAreas.forEach(programArea => queryClient.setQueryData(['programAreas', programArea.id], programArea));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)